pageserver/Cargo.toml

```diff
@@ -23,7 +23,7 @@ slog = "2.7.0"
 log = "0.4.14"
 clap = "2.33.0"
 daemonize = "0.4.1"
-tokio = { version = "1.5.0", features = ["full"] }
+tokio = { version = "1.11", features = ["process", "macros"] }
```

Comment:
I think 'macros' is only needed by the `[tokio::test]` lines in `zenith_utils/src/seqwait_async.rs:`. But `zenith_utils/src/seqwait_async.rs` is unused. We could remove `zenith_utils/src/seqwait_async.rs` altogether, and get rid of the 'macros' feature, too.

Reply:
It's a bit harder, I'm afraid, here's at least one compilation error I get when remove the feature:
https://github.com/zenithdb/zenith/blob/main/pageserver/src/walredo.rs#L609

In general, I'm surprised that only one place in all Cargo.toml require the feature specification, looks like the same feature unification in action here.